kubetest/util.go

```diff
@@ -138,9 +139,17 @@ func finishRunning(cmd *exec.Cmd) error {
 		return fmt.Errorf("error starting %v: %v", stepName, err)
 	}
 
+	sig_channel := make(chan os.Signal, 1)
+	signal.Notify(sig_channel, os.Interrupt)
+
 	finished := make(chan error)
 
 	go func() {
+		sig := <-sig_channel
+		log.Printf("Killing %v after receiving %s", -cmd.Process.Pid, sig)
+		if err := syscall.Kill(-cmd.Process.Pid, syscall.SIGKILL); err != nil {
+			fmt.Errorf("error killing %v: %v", -cmd.Process.Pid, err)
```

Comment:
I see, thanks for your help @BenTheElder 
I will remove this part like line 159.